pkg/lib/ownerutil/util.go

```diff
@@ -406,6 +406,12 @@ func InferGroupVersionKind(obj runtime.Object) error {
 			Version: apiextensionsv1beta1.SchemeGroupVersion.Version,
 			Kind:    "CustomResourceDefinition",
 		})
+	case *operatorsv1.OperatorCondition:
```

Comment:
I wonder if it's possible to get the GVK from a properly initialized `runtime.Scheme`.